definitions/src/main/scala/exercise/BuildInfo.scala

```diff
@@ -1,3 +1,8 @@
+/*
```

Comment:
Can we make the copyright headers match the same ones as the ones in Freestyle where the license is included?